tests/pages/firefox_dnt.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from .base import BasePage
+
+
+class FirefoxDNTPage(BasePage):
```

Comment:
If you're going to have a bunch of Firefox pages, you could put this in `pages/firefox/do_not_track.py`. Also, I generally prefer not to abbreviate, so I'd call this `DoNotTrackPage` under the `pages.firefox` module. In my mind this makes it more accessible to anyone unfamiliar with the DNT abbreviation.